packages/rollup-plugin-styles/src/types/sass.d.ts

```
@@ -0,0 +1,49 @@
+// eslint-disable-next-line import/no-unused-modules
+declare namespace sass {
+    type Data = Error | { contents: string } | { file: string } | null;
+
+    type Importer = (url: string, previous: string, done: (data: Data) => void) => Data | void;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)